AWSCognitoAuth/Internal/UICKeyChainStore/AWSCognitoAuthUICKeyChainStore.m

```diff
@@ -1094,6 +1087,8 @@ + (NSString *)generatePassword
 
 #pragma mark -
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-implementations"
```

Comment:
maybe a comment on which deprecated API is ignored when we have to use this would be helpful for future readers of the code, here and anywhere below that uses this directive

Reply:
will update!

Reply:
done :)